sql/core/src/main/scala/org/apache/spark/sql/Observation.scala

```diff
@@ -82,7 +86,7 @@ class Observation(name: String) {
    * @throws InterruptedException interrupted while waiting
    */
   @throws[InterruptedException]
-  def get: Row = {
+  def getAsRow: Row = {
```

Comment:
Can we just change the return type alone instead of adding other new methods?

Reply:
done, but the getAsJavaMap is still useful in two ways